.travis.yml

```diff
@@ -1,6 +1,23 @@
+# Travis CI configuration file
```

Comment:
While your updating, can you do this? http://docs.travis-ci.com/user/migrating-from-legacy/#tl%3Bdr